regression/Makefile

```diff
@@ -1,10 +1,12 @@
 
 DIRS = ansi-c \
        cbmc \
+      cbmc-typedef \
        cpp \
        cbmc-java \
        goto-analyzer \
        goto-instrument \
+       goto-instrument-typedef \
```

Comment:
For goto-instrument-typedef the chain.sh script does different things (specifically compiling with some flags to goto-cc). The cbmc ones look like they could be and I see I did agree to this before so will do.